package.json

```diff
@@ -8,7 +8,9 @@
   "dependencies": {
     "clipboard": "2.0.4",
     "primer-buttons": "2.6.1",
-    "primer-tooltips": "1.5.8"
+    "primer-tooltips": "1.5.8",
+    "webext-domain-permission-toggle": "^0.1.0",
+    "webext-dynamic-content-scripts": "^5.0.2"
```

Comment:
If you update to the latest versions the user will see a checkmark + they can disable the permission + domains in the manifest will be appear as disabled:

<img width="332" alt="" src="https://user-images.githubusercontent.com/1402241/62604954-50acb300-b923-11e9-9420-721d6dc0c60b.png">

```sh
npm i webext-dynamic-content-scripts@next webext-domain-permission-toggle@latest
```